v22.2/alter-user.md

```diff
@@ -2,26 +2,24 @@
 title: ALTER USER
 summary: The ALTER USER statement can be used to add or change a user's password.
 toc: true
-docs_area: reference.sql 
+docs_area: reference.sql
 ---
 
 The `ALTER USER` [statement](sql-statements.html) can be used to add, change, or remove a [user's](create-user.html) password and to change the role options for a user.
 
-{{site.data.alerts.callout_info}}
- Since the keywords `ROLE` and `USER` can now be used interchangeably in SQL statements for enhanced PostgreSQL compatibility, `ALTER USER` is now an alias for [`ALTER ROLE`](alter-role.html).
-{{site.data.alerts.end}}
+You can use the keywords `ROLE` and `USER` interchangeably. `ALTER USER` is an alias for [`ALTER ROLE`](alter-role.html).
 
 ## Considerations
 
 - Password creation and alteration is supported only in secure clusters.
 
 ## Required privileges
 
- To alter other users, the user must be a member of the `admin` role or have the [`CREATEROLE`](create-user.html#create-a-user-that-can-create-other-users-and-manage-authentication-methods-for-the-new-users) parameter set.
+To alter other users, the user must be a member of the `admin` role or have the [`CREATEROLE`](create-role.html#create-a-role-that-can-create-other-roles-and-manage-authentication-methods-for-the-new-roles) [role option](#role-options).
 
 ## Synopsis
 
-<div>{% include {{ page.version.version }}/sql/generated/diagrams/alter_user_password.html %}</div>
+See [Synopsis](alter-role.html#synopsis).
```

Comment:
Nit: Should this be See Alter Role instead of See Synopsis?

Reply:
I guess each one could say See *Role: Synopis, where * is Alter, Create, and Drop.